packages/api/database/migrations/1607953663025_institution_new_fields_schema.js

```diff
@@ -0,0 +1,42 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+
+const { institutionsTypes, institutionsCategories } = require('../../app/Utils');
+
+class InstitutionNewFieldsSchema extends Schema {
+	up() {
+		this.alter('institutions', (table) => {
+			// alter table
+			table.string('email');
+			table.string('phone_number');
+			table.string('website');
+			table
+				.integer('logo_id')
+				.unsigned()
+				.references('id')
+				.inTable('uploads');
```

Comment:
Coloca os onUpdate e onDelete aqui e após isso testa o que acontece quando eu excluo um upload que é `logo_id` de uma instituição.
Creio que assim funcionará 👇 
```suggestion
				.inTable('uploads')
				.onUpdate('cascade')
				.onDelete('set null');
```

Reply:
Dá certo @mateus4k, quando eu deleto o upload ele seta o campo logo_id null in institutions.